### PR TITLE
Deploy examples

### PR DESCRIPTION
**Pull request by devinmarieb**
# Description
- Add blurb about new deploy focused images
- Minor page changes to deploy section
- Add snippet to include for examples with placeholders

# Reasons
New images exist for four deployment methods

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/deploy-to-heroku.adoc [hunk 1]
```diff
@@ -23,21 +26,18 @@ link:https://www.heroku.com/[Heroku] is a popular platform for hosting applicati
 3. Use the link:https://circleci.com/developer/orbs/orb/circleci/heroku[Heroku orb] to keep your config simple. The `deploy-via-git` installs the Heroku CLI in the primary container, runs any pre deployment steps you define, deploys your application, then runs any post-deployment steps you define. See the Heroku orb page in the link:https://circleci.com/developer/orbs/orb/circleci/heroku[orbs registry] for full details of parameters and options:
 +
 ```yaml
-version: 2.1
-
+version: '2.1'
 orbs:
-    heroku: circleci/heroku@x.y # Use the Heroku orb in your config
-
```
Comment (rosieyohannan) [inline jekyll/_cci2/deploy-to-heroku.adoc:30]: Can we add the replace version warning here too? I guess we will need an adoc version of the snippet 😭

**Review by rosieyohannan — COMMENTED** [review]
Just one comment but looks great! I've used a different format for this type of warning about replacing placeholders in snippets in server docs so at some point we could brainstorm a standard and switch server docs to use this snippet too. In that case will defo need an adoc version. 🙏 🎉

File: jekyll/_cci2/deploy-to-heroku.adoc (hunk 1 above)
Comment (devinmarieb) [reply jekyll/_cci2/deploy-to-heroku.adoc:30]: oh yes thank you I actually meant to do something about this and forgot 👍

**Review by rosieyohannan — APPROVED** [review]
👍 👍 👍

**Pull request merged**